.github/workflows/dispatch-build-bottle.yml

```diff
@@ -136,13 +136,19 @@ jobs:
       - name: Setup git
         uses: Homebrew/actions/git-user-config@master
 
+      - name: Set up commit signing
+        uses: Homebrew/actions/setup-commit-signing@master
+        with:
+          signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
+
       - name: Upload and publish bottles on Bintray
         env:
           HOMEBREW_BINTRAY_USER: brewtestbot
           HOMEBREW_BINTRAY_KEY: ${{secrets.HOMEBREW_BINTRAY_KEY}}
+          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old
+          brew pr-upload --verbose --keep-old --committer='BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>'
```

Comment:
I wonder if it's better to fix this in the `Setup git` step rather than adding the `--committer` flag everywhere.

Reply:
As I understand it, we can specify a different committer using the `GIT_COMMITTER_{NAME,EMAIL}` environment variables. I went with adding the `--committer` flag is because:
* `brew` filters out `GIT_COMMITTER_{NAME,EMAIL}` from the environment.
* `Utils::Git.set_name_email!` sets the author *and* committer environment variables using `HOMEBREW_GIT_NAME` and `HOMEBREW_GIT_EMAIL`.

We would have to tell `Utils::Git.set_name_email!` one way or the other to not overwrite the environment variables if we set them in `Setup git`, so I think the `--committer` option is a more explicit alternative (compared to relying on the action to set the environment variable right).

The line does look too long though, so I could probably set the author string to an `env` variable in all these steps.

Reply:
I'm fine with `--committer` but agreed shortening the line might be nice.